apps/ngrx/7-power-of-effect/src/app/school/school.store.ts

```diff
@@ -45,4 +47,20 @@ export class SchoolStore
   ngrxOnStoreInit() {
     this.loadSchools();
   }
+
+  ngrxOnStateInit() {
+    this.effect(() =>
+      inject(PUSH_TOKEN).pipe(
```

Comment:
That's the second point I want to highly with this challenge, it's that we can use any observable inside an effect and not only the Ngrx action observbale. Effect is just a subscription. 

Nice work